src/test/java/org/takes/http/BkBasicTest.java

```diff
@@ -297,7 +350,7 @@ public void run() {
                 // @checkstyle MagicNumber (1 line)
                 final byte[] buffer = new byte[4096];
                 for (int count = input.read(buffer); count != -1;
-                    count = input.read(buffer)) {
+                     count = input.read(buffer)) {
```

Comment:
@alexey-krylov 
> @dmzaytsev you said earlier that 'okay let's keep it as is'. 

I'm sure Yegor will ask you remove the spaces. I just try to save your time

> When you do reformat code with IDEA+Yegor profile it places this line without this space?

No I didn't pull your branch,  I see only the diff